CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 Change Log
 ==========
 
+## Version 1.0.0-alpha.10
+
+_2021-03-04_
+
+* Introduce support for ViewTreeSavedStateRegistryOwner in Android container views. (#284)
+* Makes the names logged by TracingWorkflowInterceptor human readable. (#343)
+* Improve support for intercepting RenderContext calls. (#332)
+* Prefix Props & Context with "Render" (#322)
+* Introduce support for ViewTreeLifecycleOwner in Android container views. (#283)
+* Make Sink and LayoutRunner fun interfaces. (#174)
+* Import https://github.com/square/workflow-kotlin-compose (#301)
```

Comment:
This doesn't have anything to do with the release, it doesn't affect the artifacts we're releasing at all yet, so I wouldn't bother mentioning it here.